include/deal.II/grid/cell_id_translator.h

```diff
@@ -162,8 +162,8 @@ namespace internal
   CellIDTranslator<dim>::size() const
   {
     return n_coarse_cells *
-           (Utilities::pow(GeometryInfo<dim>::max_children_per_cell,
-                           n_global_levels) -
+           (Utilities::pow<types::global_cell_index>(
```

Comment:
This is definitely a bug.

Reply:
Ouch, good catch! Who would have thought that that can overflow... (Mentally, I'm thinking about this as "how many children can a cell have -- can't be that many, we'd run out of memory. But it's of course not how many we can store, but how many bits we have to allocate to address all *possible* ones. 

Nice job finding this!